test/object-entries.ts

```diff
@@ -3,9 +3,13 @@ import {expectTypeOf} from 'expect-type';
 import {objectEntries} from '../source/index.js';
 
 test('objectEntries()', t => {
-	type Entry = ['a' | 'b' | 'c', number];
-	const entries = objectEntries({a: 1, b: 2, c: 3});
+	type Entry = [1 | 'stringKey', number | string];
```

Comment:
This type is not compatible with the test case. The key `1` will never occur since it's stringified to `'1'`.

```suggestion
	type Entry = ['1' | 'stringKey', number | string];
```